pages/en/exchange-operators.mdx

```diff
@@ -0,0 +1,199 @@
+import Page from "@reason/pages/Docs";
+import DocLink from "@reason/components/DocLink";
+export default Page({ title: "Exchange Operators" });
```

Comment:
Can you change this too:

```js
export default Page({ title: "Exchange Operators | Mina Protocol", description: "Frequently asked questions about listing Mina." });
```
This will help with SEO by adding a customized title and description! :smile:

Reply:
Yes! Done.